armi/physics/neutronics/crossSectionGroupManager.py

```diff
@@ -1299,3 +1329,115 @@ def blockCollectionFactory(xsSettings, allNuclidesInProblem):
     return BLOCK_COLLECTIONS[blockRepresentation](
         allNuclidesInProblem, validBlockTypes=validBlockTypes
     )
+
+
+class LatticePhysicsUpdateFrequency(IntEnum):
```

Comment:
Do we want this class to sit in here? I think maybe more appropriately sit in the `__init__` of this package, since it's not specifically related to the xs group manager (you make heavy use of it in the lattice physics interface as well)